.gitignore

```diff
@@ -14,3 +14,4 @@ third_party/
 *~
 bazel-*
 third_party/
+dist
```

Comment:
To what I understand, if we merge [this PR](https://github.com/PaddlePaddle/Paddle/pull/1661) from @helinwang , there will be no longer directory `dist`.